docs/tutorials/ethereum/exploring-ethereum/map_contract_events_module.md

```diff
@@ -0,0 +1,121 @@
+## Retrieving Events of a Smart Contract
+
+Given a smart contract address passed as a parameter, this module returns the logs attached to the contract.
+
+### Running the Substreams
+
+First, generate the Protobuf modules and build the Rust code:
+
+```bash
+$ make protogen
```

Comment:
Take out `$` to be consistent? Especially when it's a one liner.. if there are a bunch of lines in there, and we need to show the surrounding context in `bash`, then we can leave the `$` in there.. but otherwise, simple copy&pastable stuff should elide the `$`.